packages/flutter/test/_goldens_io.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter_goldens/flutter_goldens.dart' as flutter_goldens;
```

Comment:
This can be shortened to:

```dart
export 'package:flutter_goldens/flutter_goldens.dart' show testExecutable;
```

Then you don't need to create the forwarding `testExecutable` below.

Reply:
Much better, done!